Source/SelfService/Web/theme/theme.ts

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import { createTheme } from '@mui/material/styles';
+import { TypographyOptions } from '@mui/material/styles/createTypography';
+
+const typography: TypographyOptions = {
+    fontFamily: '"Rubik", "Open sans", "Arial", sans-serif',
+    fontWeightMedium: 500,
+    h1: {
+        fontSize: '1.625rem', //26px
+        fontWeight: 800,
+    },
+    h2: {
+        fontSize: '1.5rem', //24px
+        fontWeight: 500,
+    },
+    h3: {
+        fontSize: '1.375rem', //22px
+        fontWeight: 500,
+    },
+    h4: {
+        fontSize: '1.25rem', //20px
+        fontWeight: 500,
+    },
+    h5: {
+        fontSize: '1.25rem', //20px
+        fontWeight: 300,
+    },
+    h6: {
+        fontSize: '1rem', //16px
+        fontWeight: 300,
+        textTransform: 'uppercase',
+    },
+    subtitle1: {
+        fontSize: '1.25rem', //20px
+        fontWeight: 500,
+    },
+    subtitle2: {
+        fontSize: '1.125rem', //18px
+        fontWeight: 500,
+    },
+    body1: {
+        fontSize: '1rem', //16px
+    },
+    body2: {
+        fontSize: '0.875rem', //14px
+    },
+    button: {
+        fontSize: '0.875rem', //14px
+    },
+    caption: {
+        fontSize: '0.75rem', //12px
+    },
+    overline: {
+        fontSize: '0.625rem', //10px
+    },
+};
+export const themeDark = createTheme({
+    palette: {
+        mode: 'dark',
+        primary: {
+            main: '#8C9AF8',
+            light: '#b3bbfb',
+            dark: '#6678f6',
+            contrastText: '#191A21',
```

Comment:
i've found it disappointing that MUI doesn't support storing of reused colours other than in the theme declaration where the same colours are obviously used in multiple places.
The quick solution is to copy paste the code. Mut perhaps this is another area where designers, product owners, and developers can speak the same language. And chose a name for a specific hexcolor code. This could then live in this same file  to beign with to make it easy to use as a map/js object of constant values.
Along these lines:
`const colors = {
DOLITTLE_BLACK_PRIMARY : '#191A21'
}`

This naming could then be referenced in Figma templates, asana tasks, etc...

Reply:
The goal here is not to create naming conventions that match Studio's domain, but rather from the Dolittle Design System POV.

Reply:
Oh, I really like this - the thought had come to mind as I was implementing this, but I must have lost it along the way :)

What I really like though is the additional bonus of naming the colors, so that they can be referenced in across disciplines.

I've added a follow-up task for this: https://app.asana.com/0/1201325052247030/1202282345093714/f